ister.py

```diff
@@ -384,8 +385,8 @@ def copy_os(args, template, target_dir):
     if not args.format:
         args.format = get_current_format()
     add_bundles(template, target_dir)
-    swupd_command = "swupd verify --install --path={0} " \
-                    "--manifest={1}".format(target_dir, template["Version"])
```

Comment:
ah, I may have changed that with some code I added there then removed. I can change it back.